timesketch/api/v1/resources/timeline.py

```diff
@@ -576,7 +576,7 @@ def delete(self, sketch_id: int, timeline_id: int):
             if timeline_.id != timeline_id:
                 # There are more than a single timeline using this index_name,
                 # we can't close it (unless this timeline is archived).
```

Comment:
Should we update the comment accordingly to reflect the tates that will prevent index closure?